style.css

```diff
@@ -0,0 +1,94 @@
+@import url(http://fonts.googleapis.com/css?family=Hind);
```

Comment:
This is definitely a good base stylesheet. I would like you to take these core elements and construct some core modules.